src/modules/terminalfont.c

```diff
@@ -179,6 +178,26 @@ static void printXCFETerminal(FFinstance* instance)
     }
 }
 
+static void printAlacritty(FFinstance* instance) {
+    FFstrbuf fontName;
+    ffStrbufInit(&fontName);
+    // alacritty parses config files in this order
+    ffParsePropFileConfig(instance, "alacritty/alacritty.yml", "family:", &fontName);
+    if(fontName.length == 0)
+        ffParsePropFileConfig(instance, "alacritty.yml", "family:", &fontName);
+    if(fontName.length == 0)
+        ffParsePropFileConfig(instance, ".alacritty.yml", "family:", &fontName);
+    if(fontName.length == 0)
+        ffStrbufAppendS(&fontName, "alacritty"); //by default alacritty uses it's own font called alacritty
+
+    FFfont font;
+    ffFontInitPango(&font, fontName.chars);
```

Comment:
Does alacritty really uses [pango](https://pango.gnome.org/) for font IO? This is usually a GTK thing. Pango fonts for example look like this: `Semi-Bold NotoMono Underline 10`. According to the one and only [arch wiki](https://wiki.archlinux.org/title/Alacritty) alacritty just saves the name under family, so a `ffFontInitCopy` should be much faster and enough.
The key `size: ` should only be used for font size, so we can parse for it too. You can use `ffParsePropFileConfigValues` to parse for multiple values at once.

Reply:
No it does not, I just copy-and-pasted :).
I'll rewrite to use ffFontInitCopy